.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,119 @@
+name: Build and Test
+on: push
+      
+jobs:  
+  setup_for_spack:
+    name: Run setup.py phases needed by spack
+    runs-on: ubuntu-latest
+    container: benjaminrueth/precice
```

Comment:
Shouldn't the container be `container: precice/precice`?

Reply:
Fixed in c824ac1